### DISCUSSION TITLE
Los PDF no se abren automáticamente.

### DISCUSSION THREAD

**Discussion by jolufe** [chamilo/chamilo-lms#5994] [General] [answered]
Hola.

Luego de actualizar Chamilo a la versión 1.11.28, algunos PDF que con la versión previa se abrían automáticamente dentro de una lección ahora muestran primero el botón «Descargar». Al hacer clic se abre el archivo en la pantalla.

No hice ningún cambio y son archivos que están hace tiempo en la plataforma. Solamente ocurre con algunos archivos, pero no encuentro diferencia entre los PDF que se abren automáticamente y los que no.

¿Alguna sugerencia?

SO: Windows 10

**Comment by NicoDucou** [comment, marked as answer]
Hola

Este problema ha sido reportado y solucionado en la tarea  #5917
El problema esta arreglado en la rama 1.11.30 y estara arreglado en el paquete de la version 1.11.30 cuando estara publicado proximamente.

Saludos

**Reply by jolufe** [reply]
Muchas gracias @NicoDucou.

Saludos
